src/Scrim.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const Scrim = ({ onClick, className }) => (
+  <div className={className} onClick={onClick} />
+);
+
+const StyledScrim = styled(Scrim)`
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  z-index: 10000;
```

Comment:
`follow-up`: allow for this to be passed in as props, if anyone wants to make a pr => https://www.styled-components.com/docs/basics#adapting-based-on-props